platform-dom/src/main/java/com/softicar/platform/dom/elements/time/day/AbstractDomDayPopupButton.java

```diff
@@ -9,34 +9,28 @@
 /**
  * A click-able button that shows the date of a {@link DomDayChooserDiv} object.
  * <p>
- * An object of this class already contains a {@link DomDayPopup} object. If you
- * change the date and close the pop-up menuToClose the changes you have made
- * will be shown.
+ * An object of this class already contains a {@link DomDayPopover} object. If
+ * you change the date, the popover closes and the changes will be shown.
  *
  * @author Oliver Richers
  * @author Alexander Schmidt
  */
-public abstract class AbstractDomDayPopupButton extends DomButton implements IDomInputNode {
+abstract class AbstractDomDayPopupButton extends DomButton implements IDomInputNode {
```

Comment:
I think the behavoir of this thing is great now! We should ensure that is stays like this, by adding test cases to `DomDayInputTest` (particularly, tests covering the popup).